.travis.yml

```diff
@@ -14,34 +18,46 @@ env:
 matrix:
   include:
     # current stable release w/ multisite
-    - php: "5.6"
-      env: WP_VERSION=master WP_MULTISITE=1
-    - php: "5.6"
-      env: WP_VERSION=4.5 WP_MULTISITE=0
-    - php: "5.6"
-      env: WP_VERSION=4.5 WP_MULTISITE=1
-    - php: "5.6"
-      env: WP_VERSION=4.4 WP_MULTISITE=0
-    - php: "5.5"
-      env: WP_VERSION=master WP_MULTISITE=0    
+    - php: "5.4"
+      env: WP_VERSION=4.8 WP_MULTISITE=0
     - php: "5.5"
-      env: WP_VERSION=4.5 WP_MULTISITE=0
+      env: WP_VERSION=4.8 WP_MULTISITE=0
+    - php: "5.6"
+      env: WP_VERSION=4.8 WP_MULTISITE=0
+    - php: "7.0"
+      env: WP_VERSION=4.8 WP_MULTISITE=0
+    - php: "7.1"
+      env: WP_VERSION=4.8 WP_MULTISITE=0
+    - php: "5.4"
+      env: WP_VERSION=4.8 WP_MULTISITE=1
     - php: "5.5"
-      env: WP_VERSION=4.4 WP_MULTISITE=0    
-    - php: "5.2"
-      env: WP_VERSION=master WP_MULTISITE=0
-    - php: "5.2"
-      env: WP_VERSION=4.5 WP_MULTISITE=0
-    - php: "5.2"
-      env: WP_VERSION=4.4 WP_MULTISITE=0
-    - php: "5.3"
```

Comment:
I can also understand the drop of the v 5.3, just again, could we adjust the comment around those test? I might be good to revisit the comment on lines 13-16 and adjust those accordingly.

@sboisvert what are your thoughts on deprecating 5.3 support?

Reply:
Personally I think we should drop support for < PHP 7. That being said, I don't think that's super realistic. I do think that if we think about who our target audience for these plugins, my thoughts are that they not the kind of people who don't have any control over their PHP version. I think we do need to flag it in our readme and change our docs. And I do think that we shouldn't support version older then x time and be consistent about it.
I personally think that means us testing and supporting 1 year of WordPress and whatever PHP versions are currently supported by the PHP project. We won't say no to patches that fix things for older versions but won't actively test and on them and won't officially support them. http://php.net/supported-versions.php
In the meantime I think we can safely drop 5.2 and 5.3 and strongly consider only supporting 5.6 as the earliest.

Reply:
> We won't say no to patches that fix things for older versions but won't actively test and on them and won't officially support them

In case that's the case, and in case we are going to accept patches for older versions, we should keep those unit tests on those versions, but mark them as Allowed to fail: https://docs.travis-ci.com/user/customizing-the-build/#Rows-that-are-Allowed-to-Fail

For the record, I'm fine with such approach.

Reply:
>In case that's the case, and in case we are going to accept patches for older versions, we should keep those unit tests on those versions, but mark them as Allowed to fail: https://docs.travis-ci.com/user/customizing-the-build/#Rows-that-are-Allowed-to-Fail

Good idea, I think that's the best of both worlds.